### ISSUE TITLE
[BUG] When fold_fitting_strategy is parallel_local I get error ray.exceptions.RaySystemError: System error: Unknown error

### ISSUE THREAD

**Issue by nrcjea001** [autogluon/autogluon#3462]
When fold_fitting_strategy is sequential_local, everything works fine. But when I set it to parallel_local, I get the following error.

Currently using Jupyter Notebook

```
ag_args_ensemble ={
    'fold_fitting_strategy': 'parallel_local'
}

predictor = (
    TabularPredictor(label, problem_type='regression', path=save_path, eval_metric=ag_maape_scorer)
    .fit(train_data, time_limit=time_limit, hyperparameters=hyperparameters,
         auto_stack=True, hyperparameter_tune_kwargs=hyperparameter_tune_kwargs,
         ag_args_ensemble=ag_args_ensemble)
            )
            
Warning: path already exists! This predictor may overwrite an existing predictor! path="./ag-regression-models"
Warning: hyperparameter tuning is currently experimental and may cause the process to hang.
Stack configuration (auto_stack=True): num_stack_levels=1, num_bag_folds=8, num_bag_sets=20
Beginning AutoGluon training ... Time limit = 120s
AutoGluon will save models to "./ag-regression-models"
AutoGluon Version:  0.8.3
Python Version:     3.10.12
Operating System:   Windows
Platform Machine:   AMD64
Platform Version:   10.0.22621
Disk Space Avail:   51.68 GB / 254.78 GB (20.3%)
Train Data Rows:    14476
Train Data Columns: 4
Label Column: units_sold
Preprocessing data ...
Using Feature Generators to preprocess the data ...
Fitting AutoMLPipelineFeatureGenerator...
	Available Memory:                    17379.14 MB
	Train Data (Original)  Memory Usage: 1.14 MB (0.0% of available memory)
	Inferring data type of each feature based on column values. Set feature_metadata_in to manually specify special dtypes of the features.
	Stage 1 Generators:
		Fitting AsTypeFeatureGenerator...
	Stage 2 Generators:
		Fitting FillNaFeatureGenerator...
	Stage 3 Generators:
		Fitting IdentityFeatureGenerator...
		Fitting CategoryFeatureGenerator...
			Fitting CategoryMemoryMinimizeFeatureGenerator...
	Stage 4 Generators:
		Fitting DropUniqueFeatureGenerator...
	Stage 5 Generators:
		Fitting DropDuplicatesFeatureGenerator...
	Types of features in original data (raw dtype, special dtypes):
		('float', [])  : 3 | ['lag_units_sold_1_1_groupedby_asin', 'lag_units_sold_2_1_groupedby_asin', 'lag_units_sold_3_1_groupedby_asin']
		('object', []) : 1 | ['asin']
	Types of features in processed data (raw dtype, special dtypes):
		('category', []) : 1 | ['asin']
		('float', [])    : 3 | ['lag_units_sold_1_1_groupedby_asin', 'lag_units_sold_2_1_groupedby_asin', 'lag_units_sold_3_1_groupedby_asin']
	0.1s = Fit runtime
	4 features in original data used to generate 4 features in processed data.
	Train Data (Processed) Memory Usage: 0.19 MB (0.0% of available memory)
Data preprocessing and feature engineering runtime = 0.13s ...
AutoGluon will gauge predictive performance using evaluation metric: 'root_mean_squared_error'
	This metric's sign has been flipped to adhere to being higher_is_better. The metric score can be multiplied by -1 to get the metric value.
	To change this, specify the eval_metric parameter of Predictor()
User-specified model hyperparameters to be fit:
{
	'GBM': [{'num_iterations': 100, 'max_depth': Int: lower=3, upper=11, 'learning_rate': Real: lower=0.02, upper=0.2, 'num_leaves': Int: lower=3, upper=63, 'bagging_fraction': Real: lower=0.7, upper=1, 'bagging_freq': Int: lower=1, upper=3, 'feature_fraction': Real: lower=0.5, upper=1, 'lambda_l1': Real: lower=0.0001, upper=10, 'lambda_l2': Real: lower=0.0001, upper=10, 'min_data_in_leaf': Int: lower=5, upper=50, 'min_sum_hessian_in_leaf': Real: lower=0.0001, upper=0.1}, {'num_iterations': 100, 'max_depth': -1, 'learning_rate': Real: lower=0.02, upper=0.2, 'num_leaves': Int: lower=3, upper=63, 'bagging_fraction': Real: lower=0.7, upper=1, 'bagging_freq': Int: lower=1, upper=3, 'feature_fraction': Real: lower=0.5, upper=1, 'lambda_l1': Real: lower=0.0001, upper=10, 'lambda_l2': Real: lower=0.0001, upper=10, 'min_data_in_leaf': Int: lower=5, upper=50, 'min_sum_hessian_in_leaf': Real: lower=0.0001, upper=0.1}],
	'CAT': {},
	'XGB': {},
}
AutoGluon will fit 2 stack levels (L1 to L2) ...
Fitting 4 L1 models ...
Hyperparameter tuning model: LightGBM_BAG_L1 ... Tuning model for up to 17.98s of the 119.87s of remaining time.
100%
50/50 [00:01<00:00, 32.96it/s]
	Fitting 8 child models (S1F1 - S1F8) | Fitting with ParallelLocalFoldFittingStrategy
System error: Unknown error
Traceback (most recent call last):
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\abstract\model_trial.py", line 37, in model_trial
    model = fit_and_save_model(
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\abstract\model_trial.py", line 96, in fit_and_save_model
    model.fit(**fit_args, time_limit=time_left)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\abstract\abstract_model.py", line 838, in fit
    out = self._fit(**kwargs)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\stacker_ensemble_model.py", line 165, in _fit
    return super()._fit(X=X, y=y, time_limit=time_limit, **kwargs)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\bagged_ensemble_model.py", line 266, in _fit
    self._fit_folds(
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\bagged_ensemble_model.py", line 592, in _fit_folds
    fold_fitting_strategy.after_all_folds_scheduled()
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\fold_fitting_strategy.py", line 514, in after_all_folds_scheduled
    X, y, X_pseudo, y_pseudo = self._prepare_data()
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\fold_fitting_strategy.py", line 703, in _prepare_data
    X = self.ray.put(self.X)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\ray\_private\client_mode_hook.py", line 105, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\ray\_private\worker.py", line 2452, in put
    object_ref = worker.put_object(value, owner_address=serialize_owner_address)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\ray\_private\worker.py", line 621, in put_object
    self.core_worker.put_serialized_object_and_increment_local_ref(
  File "python\ray\_raylet.pyx", line 1780, in ray._raylet.CoreWorker.put_serialized_object_and_increment_local_ref
  File "python\ray\_raylet.pyx", line 1669, in ray._raylet.CoreWorker._create_put_buffer
  File "python\ray\_raylet.pyx", line 209, in ray._raylet.check_status
ray.exceptions.RaySystemError: System error: Unknown error
	Fitting 8 child models (S1F1 - S1F8) | Fitting with ParallelLocalFoldFittingStrategy
System error: Unknown error
Traceback (most recent call last):
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\abstract\model_trial.py", line 37, in model_trial
    model = fit_and_save_model(
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\abstract\model_trial.py", line 96, in fit_and_save_model
    model.fit(**fit_args, time_limit=time_left)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\abstract\abstract_model.py", line 838, in fit
    out = self._fit(**kwargs)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\stacker_ensemble_model.py", line 165, in _fit
    return super()._fit(X=X, y=y, time_limit=time_limit, **kwargs)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\bagged_ensemble_model.py", line 266, in _fit
    self._fit_folds(
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\bagged_ensemble_model.py", line 592, in _fit_folds
    fold_fitting_strategy.after_all_folds_scheduled()
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\fold_fitting_strategy.py", line 514, in after_all_folds_scheduled
    X, y, X_pseudo, y_pseudo = self._prepare_data()
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\fold_fitting_strategy.py", line 703, in _prepare_data
    X = self.ray.put(self.X)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\ray\_private\client_mode_hook.py", line 105, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\ray\_private\worker.py", line 2452, in put
    object_ref = worker.put_object(value, owner_address=serialize_owner_address)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\ray\_private\worker.py", line 621, in put_object
    self.core_worker.put_serialized_object_and_increment_local_ref(
  File "python\ray\_raylet.pyx", line 1780, in ray._raylet.CoreWorker.put_serialized_object_and_increment_local_ref
  File "python\ray\_raylet.pyx", line 1669, in ray._raylet.CoreWorker._create_put_buffer
  File "python\ray\_raylet.pyx", line 209, in ray._raylet.check_status
ray.exceptions.RaySystemError: System error: Unknown error
	Fitting 8 child models (S1F1 - S1F8) | Fitting with ParallelLocalFoldFittingStrategy
System error: Unknown error
Traceback (most recent call last):
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\abstract\model_trial.py", line 37, in model_trial
    model = fit_and_save_model(
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\abstract\model_trial.py", line 96, in fit_and_save_model
    model.fit(**fit_args, time_limit=time_left)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\abstract\abstract_model.py", line 838, in fit
    out = self._fit(**kwargs)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\stacker_ensemble_model.py", line 165, in _fit
    return super()._fit(X=X, y=y, time_limit=time_limit, **kwargs)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\bagged_ensemble_model.py", line 266, in _fit
    self._fit_folds(
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\bagged_ensemble_model.py", line 592, in _fit_folds
    fold_fitting_strategy.after_all_folds_scheduled()
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\fold_fitting_strategy.py", line 514, in after_all_folds_scheduled
    X, y, X_pseudo, y_pseudo = self._prepare_data()
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\fold_fitting_strategy.py", line 703, in _prepare_data
    X = self.ray.put(self.X)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\ray\_private\client_mode_hook.py", line 105, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\ray\_private\worker.py", line 2452, in put
    object_ref = worker.put_object(value, owner_address=serialize_owner_address)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\ray\_private\worker.py", line 621, in put_object
    self.core_worker.put_serialized_object_and_increment_local_ref(
  File "python\ray\_raylet.pyx", line 1780, in ray._raylet.CoreWorker.put_serialized_object_and_increment_local_ref
  File "python\ray\_raylet.pyx", line 1669, in ray._raylet.CoreWorker._create_put_buffer
  File "python\ray\_raylet.pyx", line 209, in ray._raylet.check_status
ray.exceptions.RaySystemError: System error: Unknown error
	Fitting 8 child models (S1F1 - S1F8) | Fitting with ParallelLocalFoldFittingStrategy
System error: Unknown error
Traceback (most recent call last):
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\abstract\model_trial.py", line 37, in model_trial
    model = fit_and_save_model(
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\abstract\model_trial.py", line 96, in fit_and_save_model
    model.fit(**fit_args, time_limit=time_left)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\abstract\abstract_model.py", line 838, in fit
    out = self._fit(**kwargs)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\stacker_ensemble_model.py", line 165, in _fit
    return super()._fit(X=X, y=y, time_limit=time_limit, **kwargs)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\bagged_ensemble_model.py", line 266, in _fit
    self._fit_folds(
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\bagged_ensemble_model.py", line 592, in _fit_folds
    fold_fitting_strategy.after_all_folds_scheduled()
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\fold_fitting_strategy.py", line 514, in after_all_folds_scheduled
    X, y, X_pseudo, y_pseudo = self._prepare_data()
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\autogluon\core\models\ensemble\fold_fitting_strategy.py", line 703, in _prepare_data
    X = self.ray.put(self.X)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\ray\_private\client_mode_hook.py", line 105, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\ray\_private\worker.py", line 2452, in put
    object_ref = worker.put_object(value, owner_address=serialize_owner_address)
  File "C:\Users\nrcje\anaconda3\envs\ag2\lib\site-packages\ray\_private\worker.py", line 621, in put_object
    self.core_worker.put_serialized_object_and_increment_local_ref(
  File "python\ray\_raylet.pyx", line 1780, in ray._raylet.CoreWorker.put_serialized_object_and_increment_local_ref
  File "python\ray\_raylet.pyx", line 1669, in ray._raylet.CoreWorker._create_put_buffer
  File "python\ray\_raylet.pyx", line 209, in ray._raylet.check_status
ray.exceptions.RaySystemError: System error: Unknown error
```

**Comment by shchur** [comment]
I believe this bug has been fixed in the lates AG release (v1.1.1). Please feel free to re-open if that's not the case.

**Comment by Karlheinzniebuhr** [comment]
Getting the same error.


```
---------------------------------------------------------------------------
RaySystemError                            Traceback (most recent call last)
Input [In [8]](vscode-notebook-cell:?execution_count=8), in <cell line: 82>()
     [69](vscode-notebook-cell:?execution_count=8&line=69) predictor_long = TabularPredictor(
     [70](vscode-notebook-cell:?execution_count=8&line=70)     label='long_return',
     [71](vscode-notebook-cell:?execution_count=8&line=71)     eval_metric=eval_metric,
   (...)
     [78](vscode-notebook-cell:?execution_count=8&line=78)     use_bag_holdout=True,       # Added parameter
     [79](vscode-notebook-cell:?execution_count=8&line=79) )
     [81](vscode-notebook-cell:?execution_count=8&line=81) # Train predictor for 'short_return' with validation data
---> [82](vscode-notebook-cell:?execution_count=8&line=82) predictor_short = TabularPredictor(
     [83](vscode-notebook-cell:?execution_count=8&line=83)     label='short_return',
     [84](vscode-notebook-cell:?execution_count=8&line=84)     eval_metric=eval_metric,
     [85](vscode-notebook-cell:?execution_count=8&line=85)     problem_type='regression'
     [86](vscode-notebook-cell:?execution_count=8&line=86) ).fit(
     [87](vscode-notebook-cell:?execution_count=8&line=87)     train_data=train_data_short,
     [88](vscode-notebook-cell:?execution_count=8&line=88)     tuning_data=val_data_short,  # Providing validation data for early stopping
     [89](vscode-notebook-cell:?execution_count=8&line=89)     time_limit=time_limit,
     [90](vscode-notebook-cell:?execution_count=8&line=90)     presets='best_quality',
     [91](vscode-notebook-cell:?execution_count=8&line=91)     use_bag_holdout=True,        # Added parameter
     [92](vscode-notebook-cell:?execution_count=8&line=92) )
     [95](vscode-notebook-cell:?execution_count=8&line=95) # ----------------------------
     [96](vscode-notebook-cell:?execution_count=8&line=96) # Step 5: Predict on Test Set
     [97](vscode-notebook-cell:?execution_count=8&line=97) # ----------------------------
     [98](vscode-notebook-cell:?execution_count=8&line=98) 
     [99](vscode-notebook-cell:?execution_count=8&line=99) # Predict on test set for 'long_return'
    [100](vscode-notebook-cell:?execution_count=8&line=100) y_test_pred_long = predictor_long.predict(test_data_long)

File c:\ProgramData\miniforge3\envs\ML\lib\site-packages\autogluon\core\utils\decorators.py:31, in unpack.<locals>._unpack_inner.<locals>._call(*args, **kwargs)
     [28](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/core/utils/decorators.py:28) @functools.wraps(f)
     [29](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/core/utils/decorators.py:29) def _call(*args, **kwargs):
     [30](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/core/utils/decorators.py:30)     gargs, gkwargs = g(*other_args, *args, **kwargs)
---> [31](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/core/utils/decorators.py:31)     return f(*gargs, **gkwargs)

File c:\ProgramData\miniforge3\envs\ML\lib\site-packages\autogluon\tabular\predictor\predictor.py:1099, in TabularPredictor.fit(self, train_data, tuning_data, time_limit, presets, hyperparameters, feature_metadata, infer_limit, infer_limit_batch_size, fit_weighted_ensemble, fit_full_last_level_weighted_ensemble, full_weighted_ensemble_additionally, dynamic_stacking, calibrate_decision_threshold, num_cpus, num_gpus, **kwargs)
   [1093](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1093) if dynamic_stacking:
   [1094](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1094)     logger.log(
   [1095](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1095)         20,
   [1096](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1096)         f"Dynamic stacking is enabled (dynamic_stacking={dynamic_stacking}). "
   [1097](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1097)         "AutoGluon will try to determine whether the input data is affected by stacked overfitting and enable or disable stacking as a consequence.",
   [1098](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1098)     )
-> [1099](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1099)     num_stack_levels, time_limit = self._dynamic_stacking(**ds_args, ag_fit_kwargs=ag_fit_kwargs, ag_post_fit_kwargs=ag_post_fit_kwargs)
   [1101](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1101)     if (time_limit is not None) and (time_limit <= 0):
   [1102](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1102)         raise ValueError(
   [1103](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1103)             f"Not enough time left to train models for the full fit. Consider specifying a larger time_limit. Time remaining: {time_limit}s"
   [1104](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1104)         )

File c:\ProgramData\miniforge3\envs\ML\lib\site-packages\autogluon\tabular\predictor\predictor.py:1186, in TabularPredictor._dynamic_stacking(self, ag_fit_kwargs, ag_post_fit_kwargs, validation_procedure, detection_time_frac, holdout_frac, n_folds, n_repeats, memory_safe_fits, clean_up_fits, holdout_data)
   [1181](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1181)         ds_fit_kwargs["ds_fit_context"] = ds_fit_context + "/sub_fit_custom_ho"
   [1182](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1182)         logger.info(
   [1183](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1183)             f"Starting holdout-based sub-fit for dynamic stacking with custom validation data. Context path is: {ds_fit_kwargs['ds_fit_context']}."
   [1184](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1184)         )
-> [1186](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1186)     stacked_overfitting = self._sub_fit_memory_save_wrapper(
   [1187](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1187)         train_data=X,
   [1188](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1188)         time_limit=time_limit,
   [1189](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1189)         ds_fit_kwargs=ds_fit_kwargs,
   [1190](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1190)         ag_fit_kwargs=inner_ag_fit_kwargs,
   [1191](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1191)         ag_post_fit_kwargs=inner_ag_post_fit_kwargs,
   [1192](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1192)         holdout_data=holdout_data,
   [1193](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1193)     )
   [1194](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1194) else:
   [1195](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1195)     # Holdout is false, use (repeated) cross-validation
   [1196](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1196)     is_stratified = self.problem_type in [REGRESSION, QUANTILE, SOFTCLASS]

File c:\ProgramData\miniforge3\envs\ML\lib\site-packages\autogluon\tabular\predictor\predictor.py:1308, in TabularPredictor._sub_fit_memory_save_wrapper(self, train_data, time_limit, ds_fit_kwargs, ag_fit_kwargs, ag_post_fit_kwargs, holdout_data)
   [1305](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1305)     num_gpus = ResourceManager.get_gpu_count()
   [1307](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1307) # Handle expensive data via put
-> [1308](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1308) ag_fit_kwargs_ref = _ds_ray.put(ag_fit_kwargs)
   [1309](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1309) predictor_ref = _ds_ray.put(self)
   [1310](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/autogluon/tabular/predictor/predictor.py:1310) train_data_ref = _ds_ray.put(train_data)

File c:\ProgramData\miniforge3\envs\ML\lib\site-packages\ray\_private\auto_init_hook.py:24, in wrap_auto_init.<locals>.auto_init_wrapper(*args, **kwargs)
     [21](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/auto_init_hook.py:21) @wraps(fn)
     [22](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/auto_init_hook.py:22) def auto_init_wrapper(*args, **kwargs):
     [23](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/auto_init_hook.py:23)     auto_init_ray()
---> [24](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/auto_init_hook.py:24)     return fn(*args, **kwargs)

File c:\ProgramData\miniforge3\envs\ML\lib\site-packages\ray\_private\client_mode_hook.py:103, in client_mode_hook.<locals>.wrapper(*args, **kwargs)
    [101](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/client_mode_hook.py:101)     if func.__name__ != "init" or is_client_mode_enabled_by_default:
    [102](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/client_mode_hook.py:102)         return getattr(ray, func.__name__)(*args, **kwargs)
--> [103](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/client_mode_hook.py:103) return func(*args, **kwargs)

File c:\ProgramData\miniforge3\envs\ML\lib\site-packages\ray\_private\worker.py:2597, in put(value, _owner)
   [2595](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/worker.py:2595) with profiling.profile("ray.put"):
   [2596](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/worker.py:2596)     try:
-> [2597](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/worker.py:2597)         object_ref = worker.put_object(value, owner_address=serialize_owner_address)
   [2598](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/worker.py:2598)     except ObjectStoreFullError:
   [2599](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/worker.py:2599)         logger.info(
   [2600](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/worker.py:2600)             "Put failed since the value was either too large or the "
   [2601](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/worker.py:2601)             "store was full of pinned objects."
   [2602](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/worker.py:2602)         )

File c:\ProgramData\miniforge3\envs\ML\lib\site-packages\ray\_private\worker.py:704, in Worker.put_object(self, value, object_ref, owner_address)
    [696](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/worker.py:696)     raise TypeError(msg) from e
    [697](file:///C:/ProgramData/miniforge3/envs/ML/lib/site-packages/ray/_private/worker.py:697) # This *must* be the first place that we construct this python
...
File python\ray\_raylet.pyx:2831, in ray._raylet.CoreWorker._create_put_buffer()

File python\ray\_raylet.pyx:412, in ray._raylet.check_status()

RaySystemError: System error: Unknown error
```
